core/pom.xml

```diff
@@ -39,19 +40,14 @@
             <version>3.3.0.Final</version>
         </dependency>
         <dependency>
-            <groupId>org.jboss.spec.javax.batch</groupId>
-            <artifactId>jboss-batch-api_1.0_spec</artifactId>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
             <groupId>javax.inject</groupId>
             <artifactId>javax.inject</artifactId>
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>javax.enterprise</groupId>
-            <artifactId>cdi-api</artifactId>
-            <scope>provided</scope>
+            <groupId>javax.batch</groupId>
+            <artifactId>javax.batch-api</artifactId>
+            <version>${javax.batch}</version>
```

Comment:
To be more in line with what's done for other dependencies, I think we should:

 * remove the `version` tag from here
 * move the `javax.batch` property to the parent pom
 * remove the dependency management for `jboss-batch-api_1.0_spec` in the parent pom
 * add dependency management for `javax.batch-api` in the parent pom